### ISSUE TITLE
Snapshot removal issue

### ISSUE THREAD

**Issue by jellonek** [Mirantis/virtlet#135]
```
virtlet_1       | libvirt: Storage Driver error : Storage volume not found: no storage vol with matching name '/var/lib/libvirt/images/snapshot_702d1086-12bf-4398-791c-843158379044'
virtlet_1       | E1209 15:10:55.016604       1 manager.go:329] Error when removing image snapshot with path '/var/lib/libvirt/images/snapshot_702d1086-12bf-4398-791c-843158379044': Storage volume not found: no storage vol with matching name '/var/lib/libvirt/images/snapshot_702d1086-12bf-4398-791c-843158379044'
```
While domain is created with:
```
<source file='/var/lib/virtlet/snapshot_702d1086-12bf-4398-791c-843158379044'></source>
```

**Comment by jellonek** [comment]
Issue is not fixed by #151, in domain we have:
```
<source file='/var/lib/virtlet/snapshot_82bd9077-b6a7-4cd8-7160-c1e30b45b153'></source>
```
and in logs we have:
```
E1215 20:26:10.892362      36 manager.go:334] Error when removing image snapshot with path '/var/lib/virtlet/snapshot_82bd9077-b6a7-4cd8-7160-c1e30b45b153': Storage volume not found: no storage vol with matching name '/var/lib/virtlet/snapshot_82bd9077-b6a7-4cd8-7160-c1e30b45b153'
```

**Comment by jellonek** [comment]
Looks like we should call this removal with volume name instead of it's path...
